src/text/Heading.tsx

```diff
@@ -17,6 +17,7 @@ const HeaderStyleMap = {
   underlineWeighted: "text__underline-weighted",
   lightWeighted: "text__light-weighted",
   capsWeighted: "text__caps-weighted",
+  default: "",
```

Comment:
Same here. I don't think we should add this new "default" one to the Heading. If the reason you need it is for the use case of InfoCardGrid then I think it might be better to add a prop to InfoCardGrid called something like `defaultHeadingStyle` as a boolean. Then only add the "underlineWeighted" if `defaultHeadingStyle` is either false or undefined

Reply:
That makes sense. I thought of that as a potential path. Sounds good to me, I will make those changes. Thank you for the feedback!